Initium-Odp/src/com/universeprojects/miniup/server/services/MovementService.java

```diff
@@ -25,8 +25,10 @@ else if (GameUtils.equals(destinationLocationKey, pathToTake.getProperty("locati
 		
 		if (lockCode != null) 
 		{
-			if (checkHasKey(character, (long)lockCode)==false)
+			if (checkHasKey(character, (long)lockCode)==false) {
+				db.getDB().getMC().put("LongOperation-"+character.getKey(), null);
```

Comment:
I believe the problem is more a UI state than anything else. I think we simply need to refresh the page.